### PR TITLE
onboarding quay-tests component

### PR DESCRIPTION
**Pull request by LiZhang19817**
onboarding quay-tests component following guide https://docs.ci.openshift.org/docs/how-tos/onboarding-a-new-component/

### CODE REVIEW

**Comment by dongboyan77** [comment]
mark
cc @chengzhang1016

File: ci-operator/config/quay/quay-tests/.config.prowgen [hunk 1]
```diff
@@ -0,0 +1,2 @@
+private: true
```
Comment (dmage) [inline ci-operator/config/quay/quay-tests/.config.prowgen:1]: What does it give?

Comment (LiZhang19817) [reply ci-operator/config/quay/quay-tests/.config.prowgen:1]: @dmage this means allow Openshift CI to onboarding private repo, here the repo is Quay QE's test codes repo "quay-test", see https://docs.ci.openshift.org/docs/architecture/private-repositories/#openshift-priv-organization

**Comment by dmage** [comment]
@LiZhang19817 you need to fix your OWNERS files, or onboard new members to the openshift org.

**Pull request merged**